.changeset/flat-zebras-draw.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-kubernetes': minor
```

Comment:
This can be `patch` for now as it's `<1.0.0`

Reply:
```suggestion
'@backstage/plugin-kubernetes': patch
```